orchagent/qosorch.cpp

```diff
@@ -232,7 +232,7 @@ bool Dot1pToTcMapHandler::convertFieldValuesToAttributes(KeyOpFieldsValuesTuple
     sai_qos_map_list_t dot1p_map_list;
 
     // Allocated resources are freed in freeAttribResources() call
-    dot1p_map_list.list = new sai_qos_map_t[kfvFieldsValues(tuple).size()];
+    dot1p_map_list.list = new sai_qos_map_t[kfvFieldsValues(tuple).size()]();
```

Comment:
can you move this change to a separate PR?

Reply:
> can you move this change to a separate PR?

Yes.

Reply:
Fixed. The change is removed from this PR and put in a separate PR https://github.com/Azure/sonic-swss/pull/1746